hack/collector-longhorn

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+# RKE2: Logs for the kubelet are save in rke2 agent log folder
+RKE2_KUBELET_LOG_PATH=${HOST_PATH}/var/lib/rancher/rke2/agent/logs/kubelet.log
+
+function collect_sysinfo(){
+    cd ${BUNDLE_DIR}
+    mkdir -p hostinfos
+    cd hostinfos
+
+    echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+    echo "os-release : " `cat ${HOST_PATH}/etc/os-release` >> hostinfo
+    echo "system-info : " `uname -a` >> hostinfo
```

Comment:
```
> cat hostinfo
hostname :  ip-10-0-1-113
os-release :  NAME="SLES" VERSION="15-SP3" VERSION_ID="15.3" PRETTY_NAME="SUSE Linux Enterprise Server 15 SP3" ID="sles" ID_LIKE="suse" ANSI_COLOR="0;32" CPE_NAME="cpe:/o:suse:sles:15:sp3" DOCUMENTATION_URL="https://documentation.suse.com/"
system-info :  Linux ip-10-0-1-113 5.3.18-59.37-default #1 SMP Mon Nov 22 12:29:04 UTC 2021 (d10168e) x86_64 x86_64 x86_64 GNU/Linux
```
Can we have some indents and keep the line breaks instead of having everything in a single line? This could help to improve the readability. For example:
```
> echo -e "os-release :\n`cat ${HOST_PATH}/etc/os-release | sed 's/^/\t/' `" >> hostinfo
> cat hostinfo

os-release :
	NAME="SLES"
	VERSION="15-SP3"
	VERSION_ID="15.3"
	PRETTY_NAME="SUSE Linux Enterprise Server 15 SP3"
	ID="sles"
	ID_LIKE="suse"
	ANSI_COLOR="0;32"
	CPE_NAME="cpe:/o:suse:sles:15:sp3"
	DOCUMENTATION_URL="https://documentation.suse.com/"
```

Reply:
Got it, I'll work on it to improve it. thanks.